client/src/components/Dashboard.js

```diff
@@ -230,7 +230,7 @@ class Dashboard extends Component {
                       name={name}
                       userRole={userRole}
                       status={status}
-                      dueDate={dueDate}
+                      labelDueDate={labelDueDate}
```

Comment:
keep it like it was before,  dueDate={dueDate}

Reply:
I have changed it only in name column, the due date and action columns should not be affected